### PR TITLE
Removes mocked email from affiliate client method

### PR DESCRIPTION
**Pull request by fabianaferreira**
#### What is the purpose of this pull request?
Removes a mocked field in order for the developer to inform follow up email, instead of using a mocked one

#### What problem is this solving?
More flexibility when using the Affiliate client

#### How should this be manually tested?
Link on a repo such as `service-example`

#### Types of changes
- [ ] Refactor (non-breaking change that only makes the code better)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

#### Chores checklist
- [ ] Update `CHANGELOG.md`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [1.0.4] - 2020-11-09
 ### Fixed
+- Removes mocked field on a Affiliate client's method
```
Comment (lucis) [inline CHANGELOG.md:9]: It should be on `Unreleased`, right?

File: src/clients/affiliate.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ export class Affiliate extends JanusClient {
   }
 
   public registerAffiliate(
-    { name, id, salesChannelId, searchEndpoint }: AffiliateInput,
+    { name, id, salesChannelId, searchEndpoint, followUpEmail }: AffiliateInput,
```
Comment (lucis) [inline src/clients/affiliate.ts:26]: Bear in mind that this is a breaking change 💀  People using the client like it's today, if updating, will run into typing errors

Comment (lucis) [reply src/clients/affiliate.ts:26]: Should it be a major upgrade?

**Review by lucis — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (fabianaferreira) [reply CHANGELOG.md:9]: fixed!

File: src/clients/affiliate.ts (hunk 2 above)
Comment (fabianaferreira) [reply src/clients/affiliate.ts:26]: yes, true 😢

**Pull request merged**